api/questions/taxitrips.py

```diff
@@ -0,0 +1,45 @@
+
+        
+    #takes in pickup_community_area and returns most common dropoff_community_area
+    def most_common_dropoff(self, pickup_community_area):
+        query = """
+        SELECT  pickup_community_area, dropoff_community_area
+        FROM  taxitrips
+        WHERE pickup_community_area == {pickup_community_area}
+        GROUP BY pickup_community_area, dropoff_community_area
+        ORDER BY COUNT(*) desc
+        LIMIT 1
+        """.format(pickup_community_area = pickup_community_area)
+        cur = self.connection.cursor()
+        cur.execute(query)
+        rows = rows_to_dicts(cur, cur.fetchall())
+        return rows
+        
+    #gets most used payment type by pickup location
+    def get_payment_type_by_pickup(self):
```

Comment:
Nice, another interesting idea for a metric! We can review this more when you start using it, since I don't want to block launching your metric.